### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by garanews**
fix typo

### CODE REVIEW

**Review by kkraus14 — APPROVED** [review]

**Review by galipremsagar — APPROVED** [review]
@garanews Thanks for this!

I think we are okay with fixing typos in PR names in `CHANGELOG.md` as we are not altering PR numbers, but will let @kkraus14 weigh in on this.

**Comment by kkraus14** [comment]
> @garanews Thanks for this!
> 
> I think we are okay with fixing typos in PR names in `CHANGELOG.md` as we are not altering PR numbers, but will let @kkraus14 weigh in on this.

Fixing typos in the CHANGELOG is more than encouraged!

**Comment by kkraus14** [comment]
> A little sad that we are going to have to rebuild so much just because of typos, but it is good to fix them.

I can just admin merge once things are reviewed by relevant parties 😄

**Comment by kkraus14** [comment]
ok to test

**Review by ajschmidt8 — APPROVED** [review]

**Review by dillon-cullinan — APPROVED** [review]

**Comment by kkraus14** [comment]
@garanews you need to add a line to `CHANGELOG.md` in order to kick off CI.

**Comment by garanews** [comment]
@kkraus14 I can do, is enough a blank line or what?

**Comment by kkraus14** [comment]
> @kkraus14 I can do, is enough a blank line or what?

If you look at the file you'll see there's a pattern of adding a line with the pull request number and a brief description of what was done in it. So for this PR the number is 6393.

**Comment by garanews** [comment]
okay, I can add it, like under improvements of current 0.16,0 right?

**Comment by kkraus14** [comment]
> okay, I can add it, like under improvements of current 0.16,0 right?

Yup, improvements section sounds perfect, thanks!

**Comment by garanews** [comment]
@kkraus14 done!

**Review by kkraus14 — APPROVED** [review]

**Review by revans2 — APPROVED** [review]

File: cpp/include/cudf/io/csv.hpp [hunk 1]
```diff
@@ -374,7 +374,7 @@ class csv_reader_options {
   {
     if ((size != 0) and ((_skiprows != 0) or (_skipfooter != 0) or (_nrows != -1))) {
       CUDF_FAIL(
-        "When ther is valid value in skiprows or skipfooter or nrows, range size can't have "
+        "When there is valid value in skiprows or skipfooter or nrows, range size can't have "
         "non-zero value");
```
Comment (harrism) [inline cpp/include/cudf/io/csv.hpp:378]: While we are at it...

```suggestion
        "If the value of any of skiprows, skipfooter or nrows is valid, range size cannot be non-zero.");
```

File: cpp/tests/utilities_tests/type_list_tests.cpp [hunk 2]
```diff
@@ -25,7 +25,7 @@ constexpr bool is_same_v = std::is_same<T, U>::value;
 }
 
 namespace {
-// Work around to remove paranthesis surrounding a type
+// Work around to remove parenthesis surrounding a type
```
Comment (harrism) [inline cpp/tests/utilities_tests/type_list_tests.cpp:28]: Plural.

```suggestion
// Work around to remove parentheses surrounding a type
```

File: cpp/tests/utilities_tests/type_list_tests.cpp [hunk 3]
```diff
@@ -37,15 +37,15 @@ struct argument_type<T(U)> {
  * @brief Performs a compile-time check that two types are equivalent.
  *
  * @note In order to work around commas in macros, any type containing commas
- * should be wrapped in paranthesis.
+ * should be wrapped in parenthesis.
```
Comment (harrism) [inline cpp/tests/utilities_tests/type_list_tests.cpp:40]:
```suggestion
 * should be wrapped in parentheses.
```

File: cpp/tests/utilities_tests/type_list_tests.cpp [hunk 4]
```diff
@@ -37,15 +37,15 @@ struct argument_type<T(U)> {
...
  *
  * Example:
  * ```
  * EXPECT_SAME_TYPE(int, int);
  *
  * EXPECT_SAME_TYPE(int, float); // compile error
  *
- * // Paranthesis around types with commas
+ * // Parenthesis around types with commas
```
Comment (harrism) [inline cpp/tests/utilities_tests/type_list_tests.cpp:48]:
```suggestion
 * // Parentheses around types with commas
```

**Review by harrism — CHANGES_REQUESTED** [review]
Thank you. I try to encourage all RAPIDS developers to install a spell checker in their IDE, but misspellings do slip through.  Just a couple more corrections.

**Review by harrism — APPROVED** [review]

File: cpp/include/cudf/io/csv.hpp [hunk 5]
```diff
@@ -374,8 +374,7 @@ class csv_reader_options {
   {
     if ((size != 0) and ((_skiprows != 0) or (_skipfooter != 0) or (_nrows != -1))) {
       CUDF_FAIL(
-        "When there is valid value in skiprows or skipfooter or nrows, range size can't have "
-        "non-zero value");
+        "If the value of any of skiprows, skipfooter or nrows is valid, range size cannot be non-zero.");
```
Comment (harrism) [inline cpp/include/cudf/io/csv.hpp:377]: This caused the line to be too long. Run the clang-format script to fix it, or just accept this change...
```suggestion
        "If the value of any of skiprows, skipfooter or nrows is valid, range size cannot be "
        "non-zero.");
```

**Pull request merged**